roofit/histfactory/inc/RooStats/HistFactory/HistRef.h

```diff
@@ -28,37 +28,23 @@ class HistRef {
 
 
    /// constructor - use gives away ownerhip of the given pointer
-   HistRef(TH1 * h = nullptr) : fHist(h) {}
+   HistRef(TH1 * h = nullptr);
```

Comment:
Instead of moving to the cxx the code, would you mind including TH1.h in the header file?

Reply:
Sounds good. I will try it ASAP.

Reply:
It works.